examples/blueprint-construct/index.ts

```diff
@@ -121,6 +121,13 @@ export default class BlueprintConstruct {
                     controller: { service: { create: false } }
                 }
             }),
+            new blueprints.addons.KubernetesIngressAddOn({
+                crossZoneEnabled: true,
+                internetFacing: true,
+                targetType: 'ip',
+                externalDnsHostname: 'your-domain.com',
+                certificateResourceName: 'your-certificate-resource-name'
```

Comment:
How will our test work if this is set to placeholder? Please note that this blueprint is actually executed when we run do-e2e-tests. Also I am unclear how it will co-exist with the nginx addon that is from F5. 

Can you create a separate blueprint with this addon for testing in examples/examples.ts? We will need something executable before releasing.